src/helpers/validateSignUpEmailAddress.js

```diff
@@ -0,0 +1,13 @@
+import { ALPHANUMERIC } from "./validateEmailAddress";
+
+const validateSignUpEmailAddress = (email) => {
+    const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN
+
+    const EMAIL_DOMAIN_REGEX_STRING = emailDomain.replace(',','|').toUpperCase();
+    const EMAIL_VALIDATION_STRING =
+    `^[${ALPHANUMERIC}.!#$%&'*+/=?^_\`{|}~-]` + 
+    `+@${EMAIL_DOMAIN_REGEX_STRING}`
+    const EMAIL_VALIDATION_REGEX = new RegExp(EMAIL_VALIDATION_STRING);
```

Comment:
There's no reason to create the regex each time we call this function, extract it so it is only ran once.

Reply:
Oh yeah, sorry I put them in the method when I coded it with two sign up env variables and env checking. Taking them out now. 🙂

Reply:
does `const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN` go in the validateSignUpEmailAddress method or remain in it?

Reply:
ok. As discussed on slack. Regex now goes inside.